app/javascript/lib/metrics-api.js

```diff
@@ -243,11 +245,14 @@ export function log(name, props={}) {
 
   let init = Object.assign({}, defaultInit)
 
-  if (accessToken === '') {
-    // User is unauthenticated / unregistered / anonynmous
+  // only report 'authenticated' users if signed in and also registered for Terra
+  // reporting non-Terra users to Bard results in 503 errors
+  if (accessToken === '' || !registeredForTerra) {
+    // User is unauthenticated / unregistered / anonynmous / not registered for Terra
     props['distinct_id'] = userId
     delete init['headers']['Authorization']
     props['authenticated'] = false
+    props['registeredForTerra'] = registeredForTerra
```

Comment:
do we want to add another prop here like 'SCPauthenticated' to easily reflect whether the user is signed into SCP?

Reply:
Signed-in status is tracked by the `authenticated` prop.  (I've updated our [lexicon](https://mixpanel.com/report/2120588/view/19059/lexicon#transformations/event-properties) to make that easier to learn.)